samples/githubDispatch/command.js

```diff
@@ -0,0 +1,51 @@
+const { Octokit } = require('octokit')
+// argument format
+// /fusebot <user/repo> <workflow_id> <branch/tag name>
+
+// please update this with your own personal access token
+const personalAccessToken = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+
+/**
+* This is the implementation of your Fusebot command.
+* You can run it from Slack with `/fusebot trigger-fusebit-pipeline ...`.
+* Fusebot is powered by the Fusebit integration platform, https://fusebit.io/
+* 
+* Docs and samples are at https://github.com/fusebit/fusebot
+* You can talk to a fellow developer on Slack at https://fusebit.io/contact
+* Or tell us what you think with `/fusebot feedback {your-comments}` 
+*
+* Now let's build something already. 
+
+* @param ctx {FusebotContext}
+*/
+module.exports = async (ctx) => {
+    if (ctx.body.args.length === 0 || ctx.body.args[0] === "help") {
+        return help(ctx)
+    }
+    const octokit = new Octokit({ auth: personalAccessToken });
+    console.log(ctx.body.args)
+    const userRepo = ctx.body.args[0]
+    const workflowId = ctx.body.args[1]
+    const branchTagName = ctx.body.args[2]
```

Comment:
This can be done as `const [userRepo, workflowId, branchTagName] = ctx.body.args`.

Reply:
Also, add some validation and display help if the params were not specified?

Reply:
Done